app/utils/estimate_prioritization/pooling_functions.py

```diff
@@ -9,7 +9,7 @@
                           field_names = ['summary_type',
                                          'column_names',
                                          'summary_function'])
-
+#test
```

Comment:
do you need this comment @123chrisc ?

Reply:
No I don't think so! Was playing around with using doing merge requests from terminal instead of GitHub desktop. Is this comment still being actually displayed? I thought I had removed it